packages/stream_chat_flutter/lib/src/attachment_actions_modal.dart

```diff
@@ -405,7 +405,8 @@ class AttachmentActionsModal extends StatelessWidget {
       onReceiveProgress: progressCallback,
     );
     final result = await ImageGallerySaver.saveFile(filePath!);
-    downloadedPathCallback?.call((result as Map)['filePath']);
+    print('filePath: $filePath');
```

Comment:
I think we should remove the print statement

Reply:
```suggestion
```